### ISSUE TITLE
Is there no omnicc support for the Classic version of the addon? #Question #Request

### ISSUE THREAD

**Issue by Theguyxd** [casualshammy/NameplateAuras#226]
Hello again! Thanks for fixing my last issue in such a timely manner. I noticed that I can't seem to get Omnicc to work in Classic but I have no issue with it in retail. Is there no support for it in the Classic version or am I doing something wrong? Cheers again for a great addon and thanks for your time.

**Comment by rukk1** [comment]
On retail I fix this by using /fstack and finding the prefix to feed into OmniCC, but in classic it's a weird prefix that keeps changing so I dunno either.

**Comment by casualshammy** [comment]
Hello,
Please read this: https://github.com/casualshammy/NameplateAuras/issues/177#issuecomment-1384388199

**Comment by Theguyxd** [comment]
> Hello, Please read this: [#177 (comment)](https://github.com/casualshammy/NameplateAuras/issues/177#issuecomment-1384388199)

Thanks for clarifying the state of things. As of right now the built-in cooldown text is fine for me since all the customization I need can be done through editing the lua files. Examples of what I've changed is the remaining time needed for the cooldown text to change color. I have mine set to turn red at 3 sec seconds instead of 5 sec. I also changed all the colors to match up with other text colors of the ui (setting the exact rgb in the lua). Again, I did this by just editing the files a bit but those might be features someone might like. Another thing might be to turn on/off outlines and shadows for the numbers themselves. Other than that everything works as intended. Great addon as always!
